content/docs/installation/compatibility.md

```diff
@@ -3,110 +3,5 @@ title: Compatibility with Kubernetes Platform Providers
 description: 'cert-manager installation: Cloud provider compatibility'
 ---
 
-Below you will find details on various compatibility issues and quirks that you
-may be affected by when deploying cert-manager. If you believe we've missed something
-please feel free to raise an issue or a pull request with the details!
```

Comment:
After thinking more, I want to move the "compatibility" information into the "Supported releases", since we already talk about which versions of OpenShift are supported, I think it makes sense to also mention which Cloud-based Kubernetes offering are supported by cert-manager.

Reply:
I'll do that in a later PR.